src/SFA.DAS.ApplyService.Application/Apply/Roatp/RoatpPreambleQuestionBuilder.cs

```diff
@@ -368,6 +373,15 @@ public static class RoatpWorkflowQuestionTags
         public static string UKRLPPrimaryVerificationSource = "UKRLPPrimaryVerificationSource";
         public static string UKRLPVerificationCompanyNumber = "UKRLPVerificationCompanyNumber";
         public static string UKRLPVerificationCharityRegNumber = "UKRLPVerificationCharityRegNumber";
+        public static string Turnover = "Turnover";
+        public static string Depreciation = "Depreciation";
+        public static string ProfitLoss = "ProfitLoss";
+        public static string Dividends = "Dividends";
+        public static string IntangibleAssets = "IntangibleAssets";
+        public static string Assets = "Assets";
+        public static string Liabilities = "Liabilities";
+        public static string ShareholderFunds = "ShareholderFunds";
+        public static string Borrowings = "Borrowings";
```

Comment:
As mentioned. Best to prefix these in QnA so they have context in the wider context of things